experiments/src/buckets.rs

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+//! This might be where the bucketing logic can go
```

Comment:
@tarikeshaq while porting this to Rust, does it make sense to call this file `evaluator`, to mimic what https://github.com/mozilla-mobile/android-components/blob/master/components/service/experiments/src/main/java/mozilla/components/service/experiments/ExperimentEvaluator.kt does (bucketing and more)?

Reply:
I second this, if the plan is to have the bucketing and matching evaluation in this module then 'evaluator' makes more sense to me.